src/main/java/baritone/command/defaults/ETACommand.java

```diff
@@ -45,12 +46,22 @@ public void execute(String label, IArgConsumer args) throws CommandException {
             throw new CommandInvalidStateException("No process in control");
         }
         IPathingBehavior pathingBehavior = baritone.getPathingBehavior();
-        logDirect(String.format(
-                "Next segment: %.2f\n" +
-                "Goal: %.2f",
-                pathingBehavior.ticksRemainingInSegment().orElse(-1.0),
-                pathingBehavior.estimatedTicksToGoal().orElse(-1.0)
-        ));
+
+        Optional<Double> ticksRemainingInSegment = pathingBehavior.ticksRemainingInSegment();
+        Optional<Double> ticksRemainingInGoal = pathingBehavior.estimatedTicksToGoal();
+
+        if (ticksRemainingInGoal.isPresent() && ticksRemainingInSegment.isPresent()) {
```

Comment:
`ticksRemainingInSegment` shouldn't be required to show `ticksRemainingInGoal` because if planning ahead is too slow there won't be a current segment but Baritone is actually pathing and the full path eta should be shown.
I'd suggest using "N/A" as the default rather than not showing them or defaulting to -1